### PR TITLE
3.0.0-2.4.0 

### PR DESCRIPTION
**Pull request by jimmynotjim**
Release 3.0.0-2.4.0 _aka_ __beta freeze__

### Added
- Added Favicon
- New and improved primary nav (both look and interaction)
- Added expanded-state utility for getting/setting aria-expanded

### Changed
- Updated Video Code to make it usable on Events pages.
- Changed gulp JS unit testing task from `gulp:unit:js` to `gulp:unit:scripts`
- Updated Meredith Fuchs bio and images.
- Added indent rules for `var`, `let`, and `const` in ESLint config file.
- Replaced old Grunt legaccsy plugin with Gulp mq-remove plugin
- Added ability for acceptance --specs test flag to accept list of test files.
- Changes `big_radio` macro to `radio_big` and `checkbox_bg` to `checkbox_big`.
- Updated Dep Dir title to include "Acting"

### Removed
- Disables tests for landing page events, since we don't currently have events.
- Removed Ombudsman from nav for beta freeze.

### Fixed
- Fixed issue with logic displaying the Event summary state.
- Fixed missing IE only stylesheet for older systems/browsers.
- Fixed skip-navigation link for keyboard navigation.

### CODE REVIEW

**Comment by KimberlyMunoz** [comment]
Shouldn't this be a PR from refresh to beta?

**Comment by jimmynotjim** [comment]
What the heck, I thought it was! Closing

File: cfgov/core/services.py [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+class PDFGeneratorView(View):
+    render_url = None
+    stylesheet_url = None
+    filename = None
+    license = os.environ.get('PDFREACTOR_LICENSE')
+
+    def get_render_url(self):
+        if self.render_url is None:
+            raise ImproperlyConfigured(
+                "PDFGeneratorView requires either a definition of "
```
Comment (anselmbradford) [inline cfgov/core/services.py]: Trivial: Looks like it could be single quotes. Here L:33, L:40.

File: cfgov/core/tests/test_services.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        self.assertEqual(source_status, 404)
+        self.assertEqual(view.event_json, {})
+
+    def test_generate_ics(self):
+        """
+        Test that, given a specific set of JSON, the generate_ics
+        function generates valid iCalendar data.
+        """
+        view = ICSView(
+            event_source='http://localhost:9200/events/<event_slug>/')
```
Comment (anselmbradford) [inline cfgov/core/tests/test_services.py]: Use settings in `.env` https://github.com/cfpb/cfgov-refresh/blob/flapjack/.env_SAMPLE#L22-L23

File: cfgov/v1/jinja2/v1/_layouts/base.html [hunk 3]
```diff
@@ -62,20 +62,21 @@
         {% block og_article_author %}{% endblock %}
     <!-- End of Open Graph properties -->
 
-    <!-- Explicit favicon link needed for testing. Remove for production. -->
-    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
+    {# TODO: Explicit favicon link needed for testing. Remove for production. #}
+    <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico">
```
Comment (anselmbradford) [inline cfgov/v1/jinja2/v1/_layouts/base.html]: Sounds like this should be removed/updated.

File: cfgov/v1/jinja2/v1/careers/application-process/index.html [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+        %}
+        <div class="hero_wrapper hide-on_video-playing">
+            <div class="hero_card hero_card__bureau">
+                <article class="hero_preview">
+                    <div class="summary">
+                        <header class="summary_header">
+                            <h1 class="summary_heading">CFPB Hiring Video</h1>
+                        </header>
+                        <p class="summary_text">
+                            {# TODO: This is placeholder content #}
```
Comment (anselmbradford) [inline cfgov/v1/jinja2/v1/careers/application-process/index.html]: This is still placeholder content, correct?

File: cfgov/v1/jinja2/v1/careers/application-process/index.html [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+                {# TODO: Update this link when we get a transcript. #}
+                Watch the video above <a href="/transcripts/how-to-apply-for-a-federal-job-with-the-cfpb/">(or read the transcript)</a>
+                for an overview of how it works.
+            </p>
+            <p class="short-desc">
+                Please note that to ensure this process is followed,
+                we can’t take applications directly.
+                You must submit your application in one of the ways
+                described on USAJobs.
+                Here are some things to keep in mind:
```
Comment (anselmbradford) [inline cfgov/v1/jinja2/v1/careers/application-process/index.html]: This should end with a colon? cc @sarahsimpson09

File: cfgov/v1/jinja2/v1/offices/_single.html [hunk 6]
```diff
@@ -110,9 +110,9 @@ <h2 class="u-visually-hidden">
             {% endif %}
             <div class="media_body">
                 {% if resource.title %}
-                <h2 class="qa-resource-title">
+                <h3 class="h2 qa-resource-title">
```
Comment (anselmbradford) [inline cfgov/v1/jinja2/v1/offices/_single.html]: Convert to `data-qa-*` attribute.

**Pull request closed without merging**